tracer/src/Datadog.Trace.MSBuild/DatadogLogger.cs

```diff
@@ -382,14 +381,28 @@ public override void Format(StringBuilder sb, LogFormatter formatter)
                     exception: null,
                     (JsonTextWriter writer, in Context? state) =>
                     {
-                        if (state.HasValue)
+                        if (state is { } context)
                         {
                             writer.WritePropertyName("dd.trace_id");
-                            writer.WriteValue($"{state.Value.TraceId}");
+
+                            if (Tracer.Instance?.Settings?.TraceId128BitLoggingEnabled == true &&
```

Comment:
I reverted most of this change so it always injects the lower 64 bits as a decimal string, since this is CI Visibility code and we are not adding 128-bit support there yet.